File: src/Core/Houston.Application/ViewModel/ConnectorFunctionHistoryViewModels/ConnectorFunctionHistorySummaryViewModel.cs
namespace Houston.Application.ViewModel.ConnectorFunctionHistoryViewModels {
	public class ConnectorFunctionHistorySummaryViewModel {
		public Guid Id { get; set; }

		public Guid ConnectorFunctionId { get; set; }

		public string Version { get; set; } = null!;

		public BuildStatus BuildStatus { get; set; }

		public string CreatedBy { get; set; } = null!;

		public DateTime CreationDate { get; set; }

		public string UpdatedBy { get; set; } = null!;

		public DateTime LastUpdate { get; set; }
	}
}

File: src/Web/WebAngular/src/app/domain/view-models/connector-function-history-summary.view-model.ts
import { BuildStatus } from "../enums/build-status.enum";

export interface ConnectorFunctionHistorySummaryViewModel {
	id: string;
	connectorFunctionId: string;
	version: string;
	buildStatus: BuildStatus;
	createdBy: string;
	creationDate: string;
	updatedBy: string;
	lastUpdate: string;
}
